devicetypes/smartthings/qubino-flush-relay.src/qubino-flush-relay.groovy

```diff
@@ -0,0 +1,436 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Qubino Flush Relay", namespace: "qubino", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Switch"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Temperature Measurement"
```

Comment:
this device shouldn't support temperature measurement since that's entirely on the child device.